src/LodeRunner.API/src/Controllers/TestRunsController.cs

```diff
@@ -160,38 +165,65 @@ public async Task<ActionResult> CreateTestRunConfig([FromBody, SwaggerRequestBod
         /// <returns>IActionResult.</returns>
         [HttpDelete("{testRunId}")]
         [SwaggerResponse((int)HttpStatusCode.NoContent, SystemConstants.DeletedTestRun)]
-        [SwaggerResponse((int)HttpStatusCode.BadRequest, SystemConstants.InvalidTestRunId, typeof(Middleware.Validation.ValidationError), "application/problem+json")]
         [SwaggerResponse((int)HttpStatusCode.NotFound, SystemConstants.NotFoundTestRun)]
         [SwaggerResponse((int)HttpStatusCode.ServiceUnavailable, SystemConstants.TerminationDescription)]
         [SwaggerResponse((int)HttpStatusCode.InternalServerError, SystemConstants.UnableToDeleteTestRun)]
+        [SwaggerResponse((int)HttpStatusCode.InternalServerError, SystemConstants.Unknown)]

```

Comment:
@gortegaMS Is it possible to add Two message for same StatusCode in Swagger?

Reply:
I am not sure, but I do not think so